content/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax.md

```diff
@@ -74,8 +74,7 @@ Another way you can narrow down search results is to exclude certain subsets. Yo
 
 Query  | Example
 ------------- | -------------
-<code>-<em>QUALIFIER</em></code>  | **[cats stars:>10 -language:javascript](https://github.com/search?q=cats+stars%3A>10+-language%3Ajavascript&type=Repositories)** matches repositories with the word "cats" that have more than 10 stars but are not written in JavaScript.
- | **[mentions:defunkt -org:github](https://github.com/search?utf8=%E2%9C%93&q=mentions%3Adefunkt+-org%3Agithub&type=Issues)** matches issues mentioning @defunkt that are not in repositories in the GitHub organization
+<code>-<em>QUALIFIER</em></code>  | **[mentions:defunkt -org:github](https://github.com/search?utf8=%E2%9C%93&q=mentions%3Adefunkt+-org%3Agithub&type=Issues)** matches issues mentioning @defunkt that are not in repositories in the GitHub organization
```

Comment:
Adding a full stop here so that this is consistent with the other tables in the article:
```suggestion
<code>-<em>QUALIFIER</em></code>  | **[mentions:defunkt -org:github](https://github.com/search?utf8=%E2%9C%93&q=mentions%3Adefunkt+-org%3Agithub&type=Issues)** matches issues mentioning @defunkt that are not in repositories in the GitHub organization.
```